apps/mail/components/mail/mail-list.tsx

```
@@ -697,358 +459,390 @@ const Thread = memo(
+                      </button>
+                    </p>
+                  </div>
+                </div>
+              </div>
+            ) : (
+              <div className="flex flex-1 flex-col" id="mail-list-scroll">
+                <VList
+                  count={filteredItems.length}
+                  overscan={5}
+                  className="style-scrollbar flex-1 overflow-x-hidden"
+                  children={vListRenderer}
```

Biome (1.9.4)

[error] Avoid passing children using a prop

The canonical way to pass children in React is to use JSX elements

(lint/correctness/noChildrenProp)